### PR TITLE
Update rails dependencies

### PR DESCRIPTION
**Pull request by kmarszalek**
TODO:
* [x] update rails and fix
* [x] update dep and fix
* [x] ~~change login get to login post ?~~

@Marcelinna please, test search

Non-blocking new issues to be done later:
#2626
#2622

### CODE REVIEW

**Review by jswk — APPROVED** [review]
Looks good to me.
I would give heads up to @wziajka to update the ES image accordingly.

**Review by wziajka — APPROVED** [review]

**Comment by Marcelinna** [comment]
before it gets merged a fix to the eosc-portal-common has to be entered ? " change login get to login post ?"

**Comment by kmarszalek** [comment]
> before it gets merged a fix to the eosc-portal-common has to be entered ? " change login get to login post ?"
#2622

**Review by Marcelinna — APPROVED** [review]

**Comment by Marcelinna** [comment]
maybe it is problem with the seeding database for e2e tests, I have a problem locally too
image

File: app/models/offer.rb [hunk 1]
```diff
@@ -131,7 +131,6 @@ def bundled_offers_correct
       errors.add(:bundled_offers, "only non-bundled offer can have bundled offers")
     else
       errors.add(:bundled_offers, "cannot bundle self") if bundled_offers.include?(self)
-      errors.add(:bundled_offers, "cannot bundle duplicates") if duplicates?(bundled_offers)
       errors.add(:bundled_offers, "cannot bundle bundle offers") if bundled_offers.any?(&:bundle?)
```
Comment (Marcelinna) [inline app/models/offer.rb:134]: it should be removed in this PR?

Comment (kmarszalek) [reply app/models/offer.rb:134]: yeap

File: docker-compose.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ services:
       POSTGRES_USER: ${DB_POSTGRES_USER:-mp}
       POSTGRES_PASSWORD: ${DB_POSTGRES_PASSWORD:-mp}
   el:
-    image: elasticsearch:7.5.2
+    image: elasticsearch:7.17.1
     ports:
       - "127.0.0.1:9200:9200"
```
Comment (Marcelinna) [inline docker-compose.yml:19]: this seems to be the problem of not passing e2e

**Pull request merged**